src/panels/MissionsPanel/MissionContent.tsx

```diff
@@ -0,0 +1,228 @@
+  useEffect(() => {
+    if (displayCurrentPhase) {
+      setPhaseToCurrent();
+    }
+  }, [now, displayCurrentPhase, setPhaseToCurrent]);
+
+  // Whenever a phase changes, we want to get the actions that are valid for that phase
+  useEffect(() => {
+    function findCurrentActions(phase: Phase | Milestone, allActions: Action[]) {
+      return phase
+        .actions!.map((uri) => allActions.find((action) => action.identifier === uri))
+        .filter((v) => v !== undefined);
+    }
+
+    // We want to add any actions related to the whole mission
+    const phasesToCheck: (Phase | Milestone)[] = [missionOverview];
+
+    // Add any extra actions related to the currently shown phase or milestone
+    if (
+      displayedPhase.data &&
+      displayedPhase.data.actions &&
+      displayedPhase.data !== missionOverview
+    ) {
+      phasesToCheck.push(displayedPhase.data);
+    }
+
+    const result = phasesToCheck.flatMap((phase) =>
+      findCurrentActions(phase, allActions)
+    );
+
+    setCurrentActions(result);
+  }, [allActions, displayedPhase, missionOverview]);
+
+  function getTitle() {
+    // Hide title if the overview is currently shown
+    const isShowingOverview = displayedPhase.data?.name === missionOverview.name;
+    const hasType = displayedPhase.type;
+    const hideTile = isShowingOverview || !hasType;
+    return hideTile ? '' : `${displayedPhase.type}: ${displayedPhase.data.name}`;
+  }
+
+  function getTimeString() {
+    switch (displayedPhase.type) {
+      case DisplayType.Phase: {
+        const start = new Date(displayedPhase.data.timerange.start).toDateString();
+        const end = new Date(displayedPhase.data.timerange.end).toDateString();
+        return `${start} - ${end}`;
+      }
+      case DisplayType.Milestone:
+        return new Date(displayedPhase.data.date).toDateString();
+      default:
+        return '';
+    }
+  }
+
+  function setPhaseManually(phase: DisplayedPhase) {
+    setDisplayedPhase(phase);
+    setDisplayCurrentPhase(false);
+  }
+
+  return (
+    <Container my={'xs'}>
+      <Group grow wrap={'nowrap'} align={'start'}>
+        <TimeLine
+          allPhasesNested={allPhasesNested.current}
+          displayedPhase={displayedPhase}
+          missionOverview={missionOverview}
+          setDisplayedPhase={setPhaseManually}
+        />
+        <div style={{ maxWidth: 'none' }}>
+          <Group grow gap={'xs'}>
+            <Button
+              onClick={() =>
+                setPhaseManually({ type: DisplayType.Phase, data: missionOverview })
+              }
+            >
+              Overview
+            </Button>
+            <Button
+              onClick={() => setDisplayCurrentPhase((prevState) => !prevState)}
+              variant={displayCurrentPhase ? 'filled' : 'light'}
+            >
+              Current Phase
+            </Button>
+          </Group>
+          {displayedPhase.data ? (
+            <>
+              <Text>{title}</Text>
```

Comment:
This should probably be a `Title` of some sort (h2?) ? Instead of a pure text. 

For information hierarchy reasons (useful or screen readers), at least. We can style it to make it smaller later